packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -349,6 +351,59 @@ export const ObjectTestCases = {
     });
 
     this.assertText('T1T2F3');
+  },
+
+  ['@test setting the isTruthy property of an object proxy overrides its truthiness regardless of its content']() {
```

Comment:
yea so i too went back and forth on this internally. it seems like a very specious test, but this specific test case functioned as a port of an existing test (at least in spirit, if not verbatim). I'm happy to remove it altogether.